src/corppa/utils/filter.py

```diff
@@ -209,9 +244,16 @@ def main():
     filter_args.add_argument(
         "-i",
         "--idfile",
-        help="filename with list of source ids, one per line",
+        help="File containing a list of source ids (one per line) to filter to",
+        type=pathlib.Path,
         required=False,
     )
+    filter_args.add_argument(
+        "--pgfile",
+        help="CSV file containing the list of pages to filter to. File must be headerless with "
+        + "fields (1) source id and (2) digital page number.",
+        type=pathlib.Path,
```

Comment:
why headerless?

Reply:
Not a great reason, I didn't want to decided on a standardized header.

Reply:
That's a good point, I'll add it in and document.